### PR TITLE
Fixing HHVM

### PR DESCRIPTION
**Pull request by lorenzo**
* Fixed constructors signature, first argument default should have been 'now'

### CODE REVIEW

**Comment by markstory** [comment]
HHVM still doesn't like the code. I fixed a few more null values.

**Comment by lorenzo** [comment]
I fixed the tests so they all pass locally in the newest hhvm. We need to wait for travis to upgrade their stack to see them passing there

File: src/ChronosInterval.php [hunk 1]
```diff
@@ -238,30 +246,39 @@ public function __get($name)
...
+            case 'days':
+                return $this->isHHVM ? parent::__get('days') : $this->days;
+            case 'y':
+            case 'm':
+            case 'd':
+            case 'h':
+            case 'i':
+            case 's':
+            case 'invert':
+                return parent::__get($name);
```
Comment (markstory) [inline src/ChronosInterval.php]: Are these issues we should report to HHVM?

Comment (lorenzo) [inline src/ChronosInterval.php]: It is already reported, let me find the issue number and link it to this PR

**Comment by lorenzo** [comment]
@markstory this one https://github.com/facebook/hhvm/issues/2952

Apparently it is fixed in the 3.11. I tested with 3.10

**Comment by markstory** [comment]
@lorenzo Sounds like they've got it taken care of in a future release.

**Pull request merged**